### PR TITLE
proposal to create SIG-Community

### PR DESCRIPTION
**Pull request by karenhchu**

### CODE REVIEW

File: SIGs/SIG-community/proposal.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- Karen Chu (Fermyon)
+
+*As the SIG's efforts grow, we are looking for people to jump in and help with new areas.*
+
+
+## Supporting Members
+
+The following individuals support the creation of SIG-Documentation:
+
+- TBD
```
Comment (ricochet) [inline SIGs/SIG-community/proposal.md:24]:
```suggestion
- Bailey Hayes (Cosmonic)
```

File: SIGs/SIG-community/proposal.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+## Bootstrap Members
+- Divya Mohan (Suse)
+- Karen Chu (Fermyon)
+
+*As the SIG's efforts grow, we are looking for people to jump in and help with new areas.*
+
+
+## Supporting Members
+
+The following individuals support the creation of SIG-Documentation:
```
Comment (divya-mohan0209) [inline SIGs/SIG-community/proposal.md:22]: Eep also, I realized I had not amended this. Sorry Karen 😅

```suggestion
The following individuals support the creation of SIG-Community:
```

Comment (karenhchu) [reply SIGs/SIG-community/proposal.md:22]: Ooh! I totally missed that too. Thanks for catching it!

**Comment by ricochet** [comment]
The TSC approves the proposal to create SIG Community. Karen and Divya will be the initial chairs. Karen and Divya are both excellent recognized contributors working for different organizations.

The steps to bootstrap a new SIG are documented at:
https://github.com/bytecodealliance/governance/blob/main/SIGs/getting-started-guide.md#establishing-a-sig

Please work with @disquisitioner to establish the SIG. Thank you!

**Pull request merged**